src/rust-crypto/backup.ts

```diff
@@ -22,9 +22,30 @@ import { logger } from "../logger";
 import { ClientPrefix, IHttpOpts, MatrixError, MatrixHttpApi, Method } from "../http-api";
 import { CryptoEvent } from "../crypto";
 import { TypedEventEmitter } from "../models/typed-event-emitter";
+import { encodeUri } from "../utils";
 import { OutgoingRequestProcessor } from "./OutgoingRequestProcessor";
 import { sleep } from "../utils";
 
+/* eslint-disable camelcase */
```

Comment:
This is a bit heavy handed, lets not disable linters for entire files.

Reply:
fixed